package.json

```diff
@@ -55,7 +55,23 @@
         }
       },
       "prettier"
-    ]
+    ],
+    "jest": {
+      "testPathIgnorePatterns": [
```

Comment:
can you tell if this gets _added_ to the ones generated `beemo`?

Reply:
The `jest.config.js` generated by beemo doesn't have this field by default. 
This was added to avoid running the `generator` tests twice.

Reply:
sorry I wasn't clear, just wanted to see if this ignore pattern is added to or over-writes the `testPathIgnorePatterns` generated by `beemo`

Reply:
I tested removing the field from `package.json` and `jest.config.js` remove the field completely. Probably doesn't have that in base. 

I tried adding something to `package.json`'s `testMatch` yesterday and that gets **added** to the `jest.config.js` along with the one coming from `beemo`